### PR TITLE
empty commit

### PR DESCRIPTION
**Pull request by Meyclem**
## Description

This PR removes `authenticateToConnect` from `beforeEach` hook in tests otherwise they fail each time there's 2 e2e test in one file.

It seems that taiko successfully uses the `goto` but fail for a reason I don't get.

## Type of change

- [x] **Chore** (non-breaking change which refactors / improves the existing code base).
- [x] **Bug fix** (non-breaking change which fixes an issue).
- [ ] **New feature** (non-breaking change which adds functionality).
- [ ] **Breaking change** (fix or feature that would cause existing functionality to not work as expected).

## Issues

[CU-9u11zz](https://app.clickup.com/t/9u11zz)

## Checklist

- [x] My code follows the style [**contributing guidelines**][contributing_file] of this project.
- [x] I have performed a self-review of my own code.
- [ ] I have commented my code in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.

[contributing_file]: https://github.com/fewlinesco/connect-account/blob/master/README.adoc

### CODE REVIEW

**Comment by frieda-ewlines** [comment]
Task linked: [CU-9u11zz Improve the way of running e2e tests in local env](https://app.clickup.com/t/9u11zz)

**Review by Fenntasy — APPROVED** [review]
Approved but if you can update the PR name before merging 🙇‍♂️

**Pull request merged**
